src/test/java/com/vite/service/ViteHandlerTest.java

```diff
@@ -0,0 +1,16 @@
+package com.vite.service;
+
+import org.junit.Test;
+
+
+public class ViteHandlerTest {
+    /**
+     * Tests the {@link ViteHandler#ping()} function
+     * TODO add assertion about log statement, or better yet make ping return a String
+     */
+    @Test
+    public void testPing() {
+        ViteHandler viteService = new ViteHandler();
+        viteService.ping();
```

Comment:
What do you mean by "Follow up"? Is that an imperative (if so, what are you looking for? I can't push to your fork) or is that short for "see the follow up" (in which case, I don't see any)?